src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -768,13 +1038,13 @@ end
 		score = 400 -- Completed the mission with a score of 400
 	})
 	```
-	
-	For more information on progression events, refer to [GameAnalytics docs](https://docs.gameanalytics.com/integrations/sdk/roblox/event-tracking?_highlight=teleportdata#progression) on progression.
-	
+
+	For more information about progression events, refer to [GameAnalytics docs](https://docs.gameanalytics.com/integrations/sdk/roblox/event-tracking?_highlight=teleportdata#progression) on progression.
+
 	@param data ProgressionEvent
-	@return { [any]: any }
+	@return Promise
 ]=]
-function AnalyticsService:LogProgression(data: ProgressionEvent): { [any]: any }
+function AnalyticsService:LogProgression(data: ProgressionEvent)
```

Comment:
```suggestion
function AnalyticsService:LogProgression(data: ProgressionEvent): {[any]: any}
```
Outside of a custom defined promise type, {[any]: any} is the closest to a return type for a promise in my opinion.